guides/development-workflow.md

```diff
@@ -0,0 +1,341 @@
+# Development workflow
+
+## Introduction
+
+<!-- Short intro. Link to playbook for the details -->
+
+### Development principles
+
+<!-- Explain where this principles came from and how they change -->
+
+#### Communicate frequently
+
+  - space to learn from each other
+  - make shared decisions
+  - prevent duplication
+  - solve problems quicker
```

Comment:
Ah I see! So our own style guide is written with capitalised list items but at the same time references the GOV UK style guide which [specifies](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps) lower case. I'd suggest that the style-guide is the inconsistent file in all of this and not something to worry about right now.